install/install.py

```diff
@@ -61,6 +62,9 @@ def create_directory(self, directory):
     def run_command(self, command):
         print("[*] Running following command")
         print("%s"%(command))
+        if is_wget(command):
```

Comment:
``wget_wrapper`` begins with:
```python
if not is_wget(command):
    return True
```

So it does not look necessary to test it here.